administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -399,11 +404,12 @@ COM_LOCALISE_ERROR_PACKAGE_XML="The xml file is not correctly formatted"
 COM_LOCALISE_ERROR_PACKAGES_EXPORT="Error exporting the package xml. The file may not exist or you try to export reference package."
 COM_LOCALISE_EXPORT_NOT_PERMITTED="You are not authorised to export a package xml'"
 COM_LOCALISE_FIELDSET_COREPACKAGE_TRANSLATIONS="Choose only the core files translations"
+COM_LOCALISE_FIELDSET_COREPACKAGE_TRANSLATIONS_DESC="<span class='missing'>Without translation for the language selected.</span> <span class='extra'>Extra in translation for the language selected.</span>"
 COM_LOCALISE_FIELDSET_PACKAGE_DETAIL="Core Package Details"
 COM_LOCALISE_FIELDSET_PACKAGE_DETAIL_DESC="Core Package details description"
 COM_LOCALISE_FIELDSET_PACKAGE_PERMISSIONS="Permissions"
 COM_LOCALISE_FIELDSET_PACKAGE_TRANSLATIONS="Translations"
-COM_LOCALISE_FIELDSET_PACKAGE_TRANSLATIONS_DESC="Translations description"
+COM_LOCALISE_FIELDSET_PACKAGE_TRANSLATIONS_DESC="<span class='missing'>Without translation for the language selected.</span> <span class='extra'>Extra in translation for the language selected.</span>"
```

Comment:
Would have to test but what about:
<span class='missing'>Missing translation file in the selected language.</span> <span class='extra'>Extra translation file in the selected language.</span>"

Reply:
Also, I would prefer  to separate Missing and Extra with a <br>

Reply:
Done